source/support.tex

```diff
@@ -530,251 +530,141 @@
 imple\-mentation-dependent
 arithmetic types\iref{basic.fundamental}.
 
+\rSec2[version.syn]{Header \tcode{<version>} synopsis}
+
 \pnum
 The header \libheaderdef{version}
 supplies implementation-dependent information
 about the C++ standard library
 (e.g., version number and release date).
 
 \pnum
-The macros in \tref{support.ft} are defined
+The macros are defined
```

Comment:
It's not clear what macros "The macros" refers to at this point. I suggest rephrasing this sentence: "Each of the macros defined in `<version>` is also defined after inclusion of any member of the set of library headers indicated in the corresponding comment in this synopsis."

Reply:
Sounds good. Applied.